### DISCUSSION TITLE
ArmHF possible?

### DISCUSSION THREAD

**Discussion by cyrrelsneer** [mbround18/valheim-docker#210] [General]
Hi there,

I am currently owning a rather old x64 with high power consumption.
But i also have some raspberrys that I use for different docker container. 

Do you think that arm architecture will be a possibility for Valheim?

**Comment by CosmicHorrorDev** [comment]
Hello! I would love the option of running a valheim server on a pi, but currently they only provide x86_64 installations. Beyond that `steamcmd` which is the tool used for downloading and updating the installation also doesn't support arm from what I can tell.

I've seen people talk about managing it with emulation or other translation layers, but that's beyond the realm of this repo, and likely won't have very good performance.

Maybe there are some other hosting options you could try? I'm personally using an old gaming computer, but I've seen a lot of people using different cloud hosting solutions.

**Reply by cyrrelsneer** [reply]
thanks for the response.
I was hoping that due to the existence of Steam for Linux maybe it would have been possible...
Well it's not too bad, it consumes around 9€/month but still a small raspberry would have been perfect. 
Maybe one day I'll buy a x64 sbc, but well not today ;-)

Appreciate your feedback!
